commitlint.config.js

```diff
@@ -1 +0,0 @@
-module.exports = {extends: ['@commitlint/config-conventional']}
```

Comment:
Why?

Reply:
Removed it as part of https://github.com/grafana/pyroscope-nodejs/pull/76 (but not fully)